src/components/dashboard/Claim.js

```diff
@@ -213,6 +213,10 @@ const Claim = props => {
 
       if (receipt.status) {
         fireEvent(CLAIM_SUCCESS, { txhash: receipt.transactionHash })
+        const isAddedLongUseOfClaimsFeed = await userStorage.userProperties.get('isAddedLongUseOfClaimsFeed')
```

Comment:
please combine this logic and logic added in Dashboard.js in a single custom hook
also move here longUseOfClaims notification definition & count of days to count as a constant:

```
const claimDaysThreshold = 14
const claimDaysProperty = 'countClaim'

const longUseOfClaims = {
   id: '5',
   type: 'useclaim',
   status: 'completed',
   data: {
     customName: 'Congrats! You’ve made it!',
     subtitle: 'Congrats! You’ve made it!',
     receiptData: {
       from: '0x0000000000000000000000000000000000000000',
     },
     reason:
       'Nice work.\n' +
       'You’ve claimed G$ for 14 days and your spot is now secured for GoodDollar’s live launch. \n' +
       'G$ are coming your way soon!',
     endpoint: {
       fullName: 'Congrats! You’ve made it!',
     },
   },
 }

const { userProperties } = userStorage

export default () => {
  if (!Config.isPhaseZero) {
    return []
  }

  const [claimsCount, setClaimsCount] = useState(0)
  const isReachedClaimsThreshold = claimsCount >= claimDaysThreshold

  const advanceClaimsCounter = useCallback(async () => {
    const newClaimsCount = claimsCount + 1

    await userProperties.set(claimDaysProperty, newClaimsCount)

    if (claimsCount == claimDaysThreshold) {
      await userStorage.enqueueTX(longUseOfClaims)
    }
    
    setClaimsCount(count)
  }, [isReachedClaimsThreshold, setClaimsCount, claimsCount])

  useEffect(() => {
    const initializeClaimsCount = async () => {
      const count = await userProperties.get(claimDaysProperty)

      setClaimsCount(count)
    }

    initializeClaimsCount()
  })  

  return [isReachedClaimsThreshold, advanceClaimsCounter]
}
```

Reply:
then just use on Claim component. as it sends notification immediately, lines 291-300 now could be removed from Dashboard.js